crates/modelardb_server/src/parser.rs

```diff
@@ -345,6 +335,17 @@ pub fn semantic_checks_for_create_table(
             return Err(ParserError::ParserError(message.to_owned()));
         }
 
+        // Check that the table name is not a restricted keyword.
```

Comment:
Maybe change to `Check if the table name is a restricted keyword.` to match the other comments in the function.

Reply:
Fixed.

Reply:
Fixed.